packages/vue/src/components/shared/DropDown.jsx

```diff
@@ -168,6 +168,7 @@ const Dropdown = {
 															count: item.doc_count,
 															isChecked:
 																selected && this.$props.multi,
+															data: itemsToRender,
```

Comment:
Hey @tanmoyopenroot  Thanks for taking the time to fix the issue. 

It should be `item`, not `itemsToRender`.
Also, currently the `items` prop is an array of object `{ doc_count, key }` we may need to add more data to each object.

Reply:
Done, Passing only the current item